waffle/utils.py

```diff
@@ -22,7 +22,7 @@ def keyfmt(k, v=None):
         key = prefix + k
     else:
         key = prefix + hashlib.md5((k % v).encode('utf-8')).hexdigest()
-    return key.encode('utf-8')
```

Comment:
The plain ASCII restrict exists on memcache but afaik all other Django core cache backends support unicode, or can support it in the right conditions, e.g. correct database setup for dbcache. I can document it, but Django *does* already emit warnings when forming cache keys that won't work on memcached: https://github.com/django/django/blob/master/django/core/cache/backends/base.py#L226